.coveragerc

```diff
@@ -6,9 +6,11 @@ source =
     tests/unit
 omit =
     lms/scripts/*
+    lms/extensions/feature_flags/views/test.py
+    lms/views/feature_flags_test.py
 
 [report]
 show_missing = True
 precision = 2
-fail_under = 98.49
+fail_under = 98.98
```

Comment:
If we are dropping covering in this test, is it worth just fessing up?

Reply:
Naah, let us always move towards 100%. Once we hit 100% it will no longer be possible for a PR to reduce coverage by deleting more lines than it adds and stuff like that, so things will be a little more convenient